src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Figure implements Drawable {
```

Comment:
where is method for getting area?

Reply:
create an interface for this - AreaCalculator, similar to Drawable